sympy/parsing/latex/lark/transformer.py

```diff
@@ -168,7 +273,7 @@ def fraction(self, tokens):
             return "derivative", variable
         else:
             denominator = tokens[2]
-            return sympy.Mul(numerator, sympy.Pow(denominator, -1))
+            return handle_division(numerator, denominator)
```

Comment:
`handle_division` doesn't need to be an external function like this. It can be a private helper function of the `Transformer` class. The `Transformer` class already has an example of a function like this: ` _extract_differential_symbol`.